R/hmde_run.R

```diff
@@ -14,7 +14,6 @@ hmde_run <- function(model_template, ...) {
 
   # Detect model
   out <- switch(model_template$model,
-         linear = rstan::sampling(stanmodels$linear, data = model_template, ...),
```

Comment:
this branch adds a vignette, but there's also some change here to R functions and Stan files. are these intended?

Reply:
There is no 'linear' model any more, that was the first one Fonti coded up but this line remained in error.